tools/ExampleExtractor/Example.cs

```diff
@@ -91,6 +91,8 @@ private static IEnumerable<Example> LoadExamplesFromFile(string markdownFile)
             string TrimPrefix(string codeLine) =>
                 codeLine.StartsWith(prefix) ? codeLine.Substring(prefix.Length)
                         : codeLine.StartsWith(trimmedPrefix) ? codeLine.Substring(trimmedPrefix.Length)
+            // the following line handles the case of an example's blank line ("  >") inside an unnumbered list item (which is indented by 2 spaces)
```

Comment:
Please indent the comment to have the same indentation as the code.
I'd probably remove "the following line handles the case" as well - just:

// An example may be in a list, so each line starts with "  > "